### PR TITLE
Documentation fixes for AzureML minimal recipe

### PR DESCRIPTION
**Pull request by shivalikasingh95**
Hi, I recently tried out the azureml-minimal recipe and found it very useful! Thanks for creating it! Suggesting minor changes to improve the documentation based on the errors that I faced while running the recipe. Hope this is fine.
Let me know in case you'd like me to fix anything.

### CODE REVIEW

File: azureml-minimal/README.md [hunk 1]
```diff
@@ -63,8 +72,11 @@ However, ZenML works seamlessly with the infrastructure provisioned through thes
 
     ```shell
     zenml stack import <STACK-NAME> <PATH-TO-THE-CREATED-STACK-CONFIG-YAML>
+    ```
+    Now, set the stack as an active stack
```
Comment (wjayesh) [inline azureml-minimal/README.md:76]: I really like how you have separated these commands into two blocks. Makes it really easy to copy and paste!

**Review by wjayesh — APPROVED** [review]
I'm glad that you liked using the recipe, Shivalika. 
Really useful additions that make the deployment more user-friendly; thanks a lot for taking the time to create this PR.  LGTM!

**Comment by wjayesh** [comment]
As a side note: if you see any use case (or stack) that is not covered by a stack recipe, feel free to start one yourself. I can assist you all the way through. Fun fact: this recipe is also a community contribution (by @dudeperf3ct) ⚡

File: azureml-minimal/README.md (hunk 1 above)
Comment (shivalikasingh95) [reply azureml-minimal/README.md:76]: hah yes, that was the idea. Glad you noticed :)

**Comment by shivalikasingh95** [comment]
> I'm glad that you liked using the recipe, Shivalika. Really useful additions that make the deployment more user-friendly; thanks a lot for taking the time to create this PR. LGTM!

Glad you found the additions useful Jayesh! Thanks for merging the PR!

**Comment by shivalikasingh95** [comment]
> As a side note: if you see any use case (or stack) that is not covered by a stack recipe, feel free to start one yourself. I can assist you all the way through. Fun fact: this recipe is also a community contribution (by @dudeperf3ct) zap

Thanks for letting me know Jayesh! I'll definitely follow up with you on this soon! Would be happy to contribute a recipe :)

**Comment by wjayesh** [comment]
Looking forward to it, Shivalika :)

**Pull request merged**